pkg/monitoring/vmstats/collector.go

```diff
@@ -148,32 +165,65 @@ type prometheusScraper struct {
 
 func (ps *prometheusScraper) Report(vms []*k6tv1.VirtualMachine) {
 	for _, vm := range vms {
-		ps.updateVMMetrics(vm)
+		ps.updateVMInfoMetrics(vm)
+		ps.updateVMStatusMetrics(vm)
+		ps.updatePVCInfoMetrics(vm)
+	}
+}
+
+func (ps *prometheusScraper) updateVMInfoMetrics(vm *k6tv1.VirtualMachine) {
+	ps.pushMetric(metrics[infoMetric], prometheus.GaugeValue, 1,
+		vm.Name, vm.Namespace,
+		vm.Spec.Template.ObjectMeta.Annotations["vm.kubevirt.io/os"],
+		vm.Spec.Template.ObjectMeta.Annotations["vm.kubevirt.io/workload"],
+		vm.Spec.Template.ObjectMeta.Annotations["vm.kubevirt.io/flavor"],
```

Comment:
These labels are taken from the template. What will happen if they don't exist?

Reply:
there are no other "straightforward" ways to do it, we could even check the boot source, but we would have no naming or content guarantees either